Index.html

```diff
@@ -217,7 +217,11 @@ <h3>Multi-Post Stories</h3>
                     </div></div>
             </div>
         </section>
-        <div class="grid-item form"></div>
+        <section class="grid-item form">
```

Comment:
- [ ] Good job so far 👍 your page looks good and it follows the template design. It would be good to add some margin-top to this form section so that it may look more like the design.
- Your design:
![Annotation 2022-02-23 045226](https://user-images.githubusercontent.com/85113567/155259825-20c146b7-1e02-48fc-abb0-d63132142c9b.jpg)
- Figma design:
![Annotation 2022-02-23 045245](https://user-images.githubusercontent.com/85113567/155259843-057d4d6b-71d5-4b1b-aac7-e17729c0d8d8.jpg)